firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -0,0 +1,150 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import By
+from marionette.errors import (
+    NoSuchElementException,
+    StaleElementException
+)
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestSafeBrowsingNotificationBar(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.urls = [
+            # Phishing URL object = mozmill 18Feb2015
+            ("safebrowsing.notAForgeryButton.label",
+             "www.google.com/safebrowsing/report_error",
+             "https://www.itisatrap.org/firefox/its-a-trap.html"),
+            # Malware URL object  = mozmill 18Feb2015
+            ("safebrowsing.notAnAttackButton.label",
+             "www.stopbadware.org",
+             "https://www.itisatrap.org/firefox/its-an-attack.html"),
+        ]
+
+        self.prefs.set_pref('browser.safebrowsing.enabled', True)
+        self.prefs.set_pref('browser.safebrowsing.malware.enabled', True)
+
+        self.tabbar = self.browser.tabbar
+
+        # TODO? workaround for bug 1084289, mozmill: "aModule.browserWindow.maximize()
```

Comment:
Necessary for some locales where the button is not clickable inside the notification. It might be different for Marionette?